source/includes/fact-verifier-limitations.rst

```diff
@@ -0,0 +1,37 @@
+
+The embedded verifier has the following limitations:
+
+- ``mongosync`` stores the verifier state in memory, which can
+  result in a significant memory overhead. To run verification
+  checks, ``mongosync`` requires 1 GB of memory for every 20
+  million documents.
+
+- Verification cannot be resumed. If a user stops and then starts
+  ``mongosync`` for any reason, the verification process must start
+  over from the beginning.
+
+- The verifier isn't compatible with :ref:`namespace filtering
+  <c2c-filtered-sync>` or :ref:`reverse sync <c2c-api-reverse>`.
+  The :ref:`/start <c2c-api-start>` endpoint returns an error if
+  you enable these features and the verifier.
+
+- The verifier doesn't perform any verification checks on
```

Comment:
It'd be nice to make the stuff we're not verifying super obvious to readers

Reply:
Broke Limitations list down into several sections to help with visibility.